.github/workflows/continuous-integration.yml

```diff
@@ -6,6 +6,7 @@ on:
     branches:
       - '[0-9]+.[0-9]+.x'
       - 'refs/pull/*'
+      - 'renovate/*'
```

Comment:
You mentioned removing the filter instead. Would I just make this an empty array to do that?

Reply:
Just removing these 3 lines suffices

Reply:
done.